### PR TITLE
Guest List Modal added

### PR DESCRIPTION
**Pull request by ycfan23**

### CODE REVIEW

File: src/GuestList/GuestListMainPage.js [hunk 1]
```diff
@@ -1,45 +1,71 @@
...
+        <Container>
+          <h1>Betrothed</h1>
+        </Container>
+        <Container>
+          <h1>Guest List</h1>
+        </Container>
+        <Container>
+          <Link to="/dashboard">
+            <Button>Back to Dashboard</Button>
+          </Link>
```
Comment (DylanDKim) [inline src/GuestList/GuestListMainPage.js:26]: uneeded

**Review by DylanDKim — APPROVED** [review]

**Review by kyle-sablan — APPROVED** [review]

File: src/GuestList/GuestListMainPage.js [hunk 2]
```diff
@@ -1,45 +1,71 @@
...
+
+  return (
+    <>
+      <Container fluid="md" className="header">
+        <Container>
+          <h1>Betrothed</h1>
+        </Container>
+        <Container>
+          <h1>Guest List</h1>
+        </Container>
```
Comment (hithisiskhai) [inline src/GuestList/GuestListMainPage.js:22]: Still curious about containers inside of containers. Using Row would be more appropriate, functionally and semantically. Especially with single element.

File: src/GuestList/GuestListMainPage.js [hunk 3]
```diff
@@ -1,45 +1,71 @@
...
+            <Modal.Body>
+              <AddForm />
+            </Modal.Body>
+            <Modal.Footer>
+              <Button variant="secondary" onClick={handleClose}>
+                Close
+              </Button>
+              <Button variant="secondary" onClick={handleClose}>
+                Save Changes
+              </Button>
```
Comment (hithisiskhai) [inline src/GuestList/GuestListMainPage.js:43]: Two buttons not needed if they both handle the close. By default clicking outside of the model also closes it. If there is a submit button in the form itself, this save changes button is both unnecessary and inaccurate

**Review by hithisiskhai — COMMENTED** [review]
I would say buttons need to be taken a look at it in the modal. Nested container isn't breaking.

**Pull request merged**